aldryn_search/base.py

```diff
@@ -130,7 +138,30 @@ def get_description(self, obj):
         """
         return None
 
+    def check_for_app_config(self, obj):
```

Comment:
Had discussed with @czpython via Hangouts. We agreed to add another class that extends `AldrynIndexBase` with this `check_for_app_config` method.

That's to keep that layer o complexity off for non aldryn apps. So we will have `AldrynIndexBase` and `AldrynAppconfigIndexBase`.